backend/tracim_backend/config.py

```diff
@@ -364,7 +364,7 @@ def _load_global_config(self) -> None:
         self.API__KEY = self.get_raw_config("api.key", "", secret=True)
         self.SESSION__REISSUE_TIME = int(self.get_raw_config("session.reissue_time", "120"))
         default_session_data_dir = self.here_macro_replace("%(here)s/sessions_data")
-        default_session_lock_dir = self.here_macro_replace("%(here)s/sessions_data")
+        default_session_lock_dir = self.here_macro_replace("%(here)s/sessions_lock")
```

Comment:
This change can be break some existing connexion when upgrade of tracim is done? If yes, what user need to do ?

Reply:
1. It's very unlikely that some user use this config has this is the code default value but defaut config file have a value setted (basic_setup) which is different (and correct)
2. It's not a big deal if existing session are dropped.